modules/Sources/Dependencies/RestoreWalletStorage/RestoreWalletStorageInterface.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  RestoreWalletStorageInterface.swift
+//
+//
+//  Created by Lukáš Korba on 19.12.2023.
```

Comment:
```suggestion
//  Created by Lukáš Korba on 2023-12-19.
```

Reply:
Created a follow up because it affects all swift files.

https://github.com/Electric-Coin-Company/zashi-ios/issues/992